rpm/user-managerd.spec

```diff
@@ -28,7 +28,7 @@ Requires: user-managerd
 %files
 %defattr(-,root,root,-)
 %{_bindir}/%{name}
-%{_unitdir}/*.service
+%{_libdir}/systemd/system/*
```

Comment:
This should be new systemd unit dir macro.

Reply:
Now I understand what was the issue that you saw.

> %{_unitdir}/*

Will work but ```%{_unitdir}/*.service``` just having will not as there is also one .mount

Reply:
So, please change to %{_unitdir}/*